rules.md

```diff
@@ -0,0 +1,105 @@
+# Rules
+
+## Meta Overview
+
+- Asteria is a Cardano bot challenge to showcase the capabilities of the eUTxO model. In particular, it's concurrency benefits.
+- Developers can participate by building their own bots (automated agents) that interact directly with the Cardano blockchain.
+- Each bot can be defined as an off-chain process that controls particular UTxOs that are constrained by on-chain validators.
+- Once the challenge has been layed out by the administrators, all following interactions and rewards are fully decentralized.
+
+## Game Overview
+
+- Your bot is in charge of controlling a (space) ship that moves in a 2D grid.
+- The goal is to reach coordinates (0, 0) allowing the rewards of the challenge to be claimed.
+- Your movement is constrained by a maximum speed and the availability of a fuel resource.
+- Fuel is freely available at fixed coordinates in the grid, ships can gather the fuel if their coordinates overlap.
+
+## Components
+
+### Grid
+
+- the grid is a virtual space in 2 dimensions (`x`, `y`).
+- there's no on-chain representation of the grid
+- coordinates on each dimension are discrete (integer values)
+- the coordinates `(0, 0)` are considered the center of the grid
+
+### Ship
+
+- a ship is identified by a Cardano native asset token of a specific policy id. We call this class of token `ShipToken`.
+- the UTxO that holds the `ShipToken` defines the state of the ship. We call this class of UTxOs `ShipState`.
+- the datum of the `ShipState` UTxO contains information about current coordinates and available fuel.
+- all `ShipUtxo` are locked in a particular script address that constrains the changes that can be applied to the state.
+
+```js
+// ShipState
+{
+    assets: {
+        [SHIP_MINT_POLICY]: {
+            [$ShipName]: 1
+        }
+    },
+    datum: {
+        pos_x: Int, // x coordinate of the current position
+        pos_y: Int, // y coordinate of the current position
+        pos_ts: PosixTime, // timestamp of last movement
+        fuel: Int // total amount of available fuel
+    }
+}
+```
+
+> ::question::
+> should we handle fuel as part of the datum or rely on a native token? 
```

Comment:
I propose to represent the fuel as an integer in the datum. Both for ship status and fuel pellet status.